rest_framework/fields.py

```diff
@@ -873,20 +875,28 @@ def to_internal_value(self, data):
 
 # Number types...
 
-class IntegerField(Field):
+
+class NumberField(Field):
```

Comment:
The change footprint of this pull request is fairly large given that it introduces a new base class here. I'd be interested in seeing the equivalent pull request without introducing a new base class. We could then consider if we want a `NumberField` as a separate feature request, since that's really a different issue to numeric localisation support.